javascript/sdk/src/index.ts

```diff
@@ -16,19 +15,18 @@ interface ForeverVMOptions {
 }
 
 export class ForeverVM {
-  baseUrl = 'https://api.forevervm.com'
+  #token = process.env.FOREVERVM_TOKEN || ''
+  #baseUrl = 'https://api.forevervm.com'
 
-  constructor(
-    private token: string,
-    options: ForeverVMOptions = {},
-  ) {
-    if (options.baseUrl) this.baseUrl = options.baseUrl
+  constructor(token: string, options: ForeverVMOptions = {}) {
+    this.#token = token
+    if (options.baseUrl) this.#baseUrl = options.baseUrl
   }
 
-  private async getRequest(path: string) {
-    const response = await fetch(`${this.baseUrl}${path}`, {
+  async #get(path: string) {
+    const response = await fetch(`${this.#baseUrl}${path}`, {
       headers: {
-        Authorization: `Bearer ${this.token}`,
+        Authorization: `Bearer ${this.#token}`,
```

Comment:
```suggestion
        'Authorization': `Bearer ${this.#token}`,
```

Reply:
ah i see what you mean. in javascript there's no difference between these two. we'd need to do something like the below:
```suggestion
        "'Authorization'": `Bearer ${this.#token}`,
```

Reply:
ah i see what you mean. that actually involves setting [this prettier option](https://prettier.io/docs/en/options#quote-props) and formatting the codebase rather than making any manual changes here

Reply:
ah I was wondering if maybe prettier was doing this but I would have thought `preserve` would be the default. Ok, I'm fine with as-is.